apps/aens/src/aens_utils.erl

```diff
@@ -59,13 +60,13 @@ check_claimed_status(Name) ->
         revoked -> {error, name_revoked}
     end.
 
+name_parts(Name) ->
+    binary:split(Name, ?LABEL_SEPARATOR, [global, trim]).
+
 validate_name(Name) ->
-    case binary:split(Name, ?LABEL_SEPARATOR, [global, trim]) of
-        [_Label, RegistrarNS] ->
-            case [RN || RN <- aec_governance:name_registrars(), RN =:= RegistrarNS] of
-                [] -> {error, registrar_unknown};
-                _  -> ok
-            end;
+    case name_parts(Name) of
+        [_Label, _RegistrarNS] ->
+            ok;
```

Comment:
Why do we no longer need to validate the name?

Reply:
We do, but only validate what is possible to check without additional height parameter.

The registrar check which was removed here is now validated here:  https://github.com/aeternity/aeternity/blob/PT-167889960-disallow-claiming-test-registrar/apps/aeprimop/src/aeprimop.erl#L1619

Reply:
It would not be a lot of work to actually here check the registrats from all possible protocol heights.... could even make that a aec_governance function. Then it is super easy to be backward compatible and have an extra check in process transaction